example_test.go

```diff
@@ -397,6 +397,13 @@ func ExampleScheduler_Monday() {
 	// Monday
 }
 
+func ExampleScheduler_MonthLastDay() {
```

Comment:
the alphabetize check is complaining that this should go underneath the Month example on 407